services/gossip/adapter/tcp/direct_transport_roundtip_metric_test.go

```diff
@@ -0,0 +1,277 @@
+//+build !race
+// Histogram metrics are disabled in race mode
+
+package tcp
+
+import (
+	"context"
+	"fmt"
+	"github.com/orbs-network/orbs-network-go/services/gossip/adapter"
+	"github.com/orbs-network/orbs-network-go/test"
+	"github.com/orbs-network/orbs-network-go/test/with"
+	"github.com/orbs-network/orbs-spec/types/go/primitives"
+	"github.com/orbs-network/orbs-spec/types/go/protocol/gossipmessages"
+	"github.com/stretchr/testify/require"
+	"net"
+	"regexp"
+	"testing"
+	"time"
+)
+
+func TestDirectTransport_SupportsBroadcastTransmissions(t *testing.T) {
```

Comment:
isn't this a duplication from `direct_transport_test`?

Reply:
It was moved here accidentally, will move back